clang/docs/SanitizerSpecialCaseList.rst

```diff
@@ -77,10 +77,8 @@ For example, supplying the above ``ignorelist.txt`` to
 ``-fsanitize-ignorelist=ignorelist.txt`` disables overflow sanitizer
 instrumentation for arithmetic operations containing values of type ``int``.
 
-The ``=sanitize`` category is also supported. Any types assigned to the
-``sanitize`` category will have their sanitizer instrumentation remain. If the
-same type appears within or across ignorelists with different categories the
-``sanitize`` category takes precedence -- regardless of order.
+The ``=sanitize`` category is also supported. Any entries assigned to the
```

Comment:
Match by entry ``=sanitize`` category enables sanitizer instrumentation, even if it was ignored by entries before.